iothub/service/src/Http/HttpMessageHelper.cs

```diff
@@ -85,6 +76,18 @@ internal static async Task<T> DeserializeResponseAsync<T>(HttpResponseMessage re
             return JsonConvert.DeserializeObject<T>(str);
         }
 
+        /// <summary>
```

Comment:
Not a public method so documenting it is only for code readers, mainly us.

All upstream methods that are public need to have these exceptions documented, however.